go/tools/builders/compilepkg.go

```diff
@@ -76,8 +77,9 @@ func compilePkg(args []string) error {
 	fs.StringVar(&nogoPath, "nogo", "", "The nogo binary. If unset, nogo will not be run.")
 	fs.StringVar(&packageListPath, "package_list", "", "The file containing the list of standard library packages")
 	fs.StringVar(&coverMode, "cover_mode", "", "The coverage mode to use. Empty if coverage instrumentation should not be added.")
-	fs.StringVar(&outPath, "o", "", "The output archive file to write compiled code")
-	fs.StringVar(&outFactsPath, "x", "", "The output archive file to write export data and nogo facts")
+	fs.StringVar(&outPath, "o", "", "The full output archive file required by the linker")
+	fs.StringVar(&outXPath, "x", "", "The export-only output archive required to compile dependent packages")
```

Comment:
In my head, the flags here supposed to be reverse:

Exported files, or ‘x’, are needed by the linker as it let the linker know which object is being exported from which package to “link” the final binary.

Perhaps, to avoid future confusion, we should use “-o” and “-lo” for “object” and “linkobject” instead of “-o” and “-x”. The precedent for this convention is in Go source:

https://cs.opensource.google/go/go/+/master:test/linkobj.go;l=88;drc=b2fd76ab8dd72b9f2135c1e9b15130e6028fc838

Wdyt?

Reply:
Sounds good, I also renamed some variables.